### PR TITLE
Pull in optional 0.9 API fixes

### PR DESCRIPTION
**Pull request by skitt**
A couple of non-optional entries were added to the Submariner CRD,
which breaks upgrades. This updates to a fixed version of Submariner.

Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

**Review by mangelajo — APPROVED** [review]

**Comment by skitt** [comment]
*THIS PR CAN’T BE MERGED AS-IS*. Its purpose is to verify https://github.com/submariner-io/submariner/pull/1246; this PR will need to be updated once that PR is merged (to use the merged hash).

**Comment by mangelajo** [comment]
> _THIS PR CAN’T BE MERGED AS-IS_. Its purpose is to verify [submariner-io/submariner#1246](https://github.com/submariner-io/submariner/pull/1246); this PR will need to be updated once that PR is merged (to use the merged hash).

We have a chicken egg here ;) we may need to merge this one, then the other. Otherwise all E2E on the other is red.

**Comment by skitt** [comment]
OK let’s merge this, if it passes, and I’ll submit a follow-up PR to adjust the hash.

**Comment by mangelajo** [comment]
Thanks, as we were speaking on the slack, probably a good solution to this in the future would be re-compiling the operator in the submariner tree when there are changes to the submariner api.

**Review by tpantelis — APPROVED** [review]

**Pull request merged**
